collector/src/main/java/com/bitmovin/analytics/BitmovinAnalyticsConfig.java

```diff
@@ -30,6 +30,7 @@
     private Boolean ads = true;
     private Context context;
     private Boolean isLive;
+    private Boolean resendDataOnHttpTimeout = false;
```

Comment:
Not sure about the name of this config, as it's really specific und would be hard to change later (without breaking changes). Do we want to have that more generic, what are our plans for this feature in the future? (in regards to supporting longer offline sessions?) @Tigraine

Reply:
Yeah the name is not great - also it should go into the `CollectorConfig` that's specifically for configuring the behavior of the collector